### PR TITLE
COMMUNITY DATA: Include Community Feedback

### LINKED ISSUE #34: COMMUNITY DATA: Include community feedback
After the event in Amsterdam, we received some community feedback from @danielskatz in our Google Doc. Great suggestions! We will include his feedback in this next round of work.

https://docs.google.com/document/d/1zUVaKBOXTIPRWDnUkBU0SIARjwS-p7mvKUr7-1qWfb0/edit#heading=h.pdb40h2oo2he

### PR DESCRIPTION
**Pull request by Ruv7**
Quick update to Include comments from @danielskatz  to improve text. Many thanks for your feedback and feel free to add further comments now that a big chunk of time has passed. Also, a shout out to @Anavelyz for helping me to submit this PR.

Closes #34

### CODE REVIEW

**Review by danielskatz — APPROVED** [review]
lgtm

File: unconference/methodologies_collecting_data.md [hunk 1]
```diff
@@ -45,16 +45,17 @@ What can be done with results?
...
+CONTRIBUTORS: People who contribute to an OS project. Contributions can be code, non code, or both. Contributors can also be users but for the purposes of this survey we’re asking them to evaluate their experience as Contributors. 
  - Code contributor - a person who opens a pull request
  - Non-code contributor
-   - Organizers of events, meetings, conferences
+   - Organizers/speakers for events, meetings, conferences
    - Writing, editing and updating documentation
    - UI or UX Design
-   - Operations activities that move forward the goals of a project, such as but not limited to: social media administration, communications administration, fundraising, governance
+   - Testing
+   - Operations activities that move forward the goals of a project, such as but not limited to: project management, social media administration, communications administration, fundraising, governance
```
Comment (OriolAbril) [inline unconference/methodologies_collecting_data.md:55]: I get kind of a duality feeling from reading this which I don't think should be there. At least to me (someone who is known to be a bit too "cuadriculado"), I perceive a first level/split: code vs non-code, then the non-code half has further subdivisions. This leaves me with the impression that code and non-code are equiparable, thus, all non-code contributions _together_ can be compared/comparable to code contributions. I also get the "contributions aren't only code" message and "all contributions are valuable" which imo deserve more emphasis.

An idea to try and emphasize more these last two messages could be to remove mentions to code/non-code and to have a single level bullet point list. The content would be basically the same:

```
CONTRIBUTORS: People who contribute to an OS project. CONTRIBUTIONS ARE NOT LIMITED TO CODE. Contributors can also be users but for the purposes of this survey we’re asking them to evaluate their experience as Contributors.
 - Code contributor - a person who opens a pull request
 - Organizers/speakers for events, meetings, conferences
 - Writing, editing and updating documentation
 - UI or UX Design
 - Testing
 - Operations activities that move forward the goals of a project, such as but not limited to: project management, social media administration, communications administration, fundraising, governance
```

Comment (Anavelyz) [reply unconference/methodologies_collecting_data.md:55]: Your suggestion makes sense, thank you for sharing it!

File: unconference/methodologies_collecting_data.md [hunk 2]
```diff
@@ -45,16 +45,17 @@ What can be done with results?
  - Compare year over the year NPS, create dashboard (pending - can we create a dashboard)
 
 #### Community: Targeted Respondents
-CONTRIBUTORS: People who contribute to an OS project, this can be code, documentation or both. Contributors can also be users but for the purposes of this survey we’re asking them to evaluate their experience as Contributors. 
+CONTRIBUTORS: People who contribute to an OS project. Contributions can be code, non code, or both. Contributors can also be users but for the purposes of this survey we’re asking them to evaluate their experience as Contributors.
```
Comment (Anavelyz) [inline unconference/methodologies_collecting_data.md:48]:
```suggestion
CONTRIBUTORS: People who contribute to an OS project. CONTRIBUTIONS ARE NOT LIMITED TO CODE. Contributors can also be users but for the purposes of this survey we’re asking them to evaluate their experience as Contributors.
```

File: unconference/methodologies_collecting_data.md (hunk 1 above)
Comment (Ruv7) [reply unconference/methodologies_collecting_data.md]: Oriol - saludos, I hope all is well with you! I was away from Jupyter for a few months, apologies for the delay here. I think this is a great suggestion and feedback. Anavelyz and I worked on this today and have made some changes based on your comments. If you have time to take a look we have requested your review.

**Review by OriolAbril — APPROVED** [review]

**Comment by Ruv7** [comment]
@kamila-NF Anavelyz and I continue making progress on our DISC Unconference 2023 project! We have a question on the structure/review needed for moving forward for our work. This is something that was left undefined during the Unconference event. Who approves changes we make to the work that was created at the event? We have permission to make the changes in GH but we are not sure that we should. We have reached out to our teammates from the event but are not hearing back and it's been several weeks. Perhaps we should create a fork? Maybe this is a question we should ask the DISC Committee? This is in addition to the question we have about wanting to connect with anyone who may have experience conducting an NPS survey.  Please advise what you think is best.

**Comment by kamila-NF** [comment]
@Ruv7 @Anavelyz @melissawm  Discussing the subject during the DISC Committee meeting sounds reasonable. The next meeting is planned for June 13th. I'm happy to add this point to the agenda.

In my opinion, since you haven't received a reply from other team members for a couple of months, it's fair that you continue making changes in the main repo. 

NPS - working on it. Will keep you posted.

> @kamila-NF @melissawm - I wanted to share here that @Anavelyz andI continue making progress on our DISC Unconference 2023 project. We have a question on the structure/review needed for moving forward for our work. This is something that was left undefined during the Unconference event in Amsterdam.
> 
> Who approves changes we make to the work that was created at the event? We have permission to make the changes in GH but we are not sure that we should. We have reached out to our teammates from the event but are not hearing back and it's been several weeks. Perhaps we should create a fork? Maybe this is a question we should ask the DISC Committee? This is in addition to the question we have about wanting to connect with anyone who may have experience conducting an NPS survey.  Conversations are happening in a few places and we wanted to be sure to add here for archival reasons.
> 
> Please advise what you think is best.

**Comment by kamila-NF** [comment]
@Ruv7 @Anavelyz @melissawm We discussed the matter at the DISC Committee meeting today. In general, all the actions need to be in line with the licence. All the project members can make a change. It is in a good tone to discuss changes with all the team members; however, if the team members are not responding for a significant time, the interpretation would be that they abstain from participation in this decision. In short words - feel free to approve the changes. 

NPS—You have recived two introductions. I will let you know if we have any other volunteers to contribute. 

> @Ruv7 @Anavelyz @melissawm Discussing the subject during the DISC Committee meeting sounds reasonable. The next meeting is planned for June 13th. I'm happy to add this point to the agenda.
> 
> In my opinion, since you haven't received a reply from other team members for a couple of months, it's fair that you continue making changes in the main repo.
> 
> NPS - working on it. Will keep you posted.
> 
> > @kamila-NF @melissawm - I wanted to share here that @Anavelyz andI continue making progress on our DISC Unconference 2023 project. We have a question on the structure/review needed for moving forward for our work. This is something that was left undefined during the Unconference event in Amsterdam.
> > Who approves changes we make to the work that was created at the event? We have permission to make the changes in GH but we are not sure that we should. We have reached out to our teammates from the event but are not hearing back and it's been several weeks. Perhaps we should create a fork? Maybe this is a question we should ask the DISC Committee? This is in addition to the question we have about wanting to connect with anyone who may have experience conducting an NPS survey.  Conversations are happening in a few places and we wanted to be sure to add here for archival reasons.
> > Please advise what you think is best.

**Comment by Ruv7** [comment]
Thanks so much! This is super helpful.

**Pull request merged**